next/utils/forms.tsx

```diff
@@ -0,0 +1,415 @@
+import Form from '@rjsf/core'
```

Comment:
why the .tsx rename ? I couldn't find any jsx within the file, you should be able to keep it as forms.ts as before (would prefer doing that)

I've skipped over this file for now as it's quite hard to track the changes within github ui with renamed files

Reply:
i forgot to rename it back to ts. I was just trying one trick few few weeks ago. Thanks !